### PR TITLE
Exercise-5

### PR DESCRIPTION
**Pull request by hennigk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,84 @@
...
+    lastName: "Hennig",
+    addresses: [
+      {type: "work", address: "13 wallaby lane"}]
+  }
+}
+
+var count = 0;
+for (var key in entries) {
+  count +=1;
+}
```
Comment (ziad-saab) [inline index.js]: @hennigk Easier and more robust way to do the same thing:
```javascript
var count = Object.keys(entries).length;
```

Comment (ziad-saab) [inline index.js]: Actually @hennigk it seems you're not even using `count`...

**Comment by hennigk** [comment]
I think I copied my code from ex-4 and forgot to remove the count variable. 
but I will update ex-4 with your suggestion.

**Comment by ziad-saab** [comment]
@hennigk :+1:

**Pull request closed without merging**